packages/wrangler/src/init.ts

```diff
@@ -620,3 +706,78 @@ async function findPath(
 		});
 	}
 }
+
+async function createWorkerConfig(
+	accountId: string,
+	fromDashScriptName: string,
+	environment = "production"
```

Comment:
since we're fetching the value for environment, maybe we shouldn't accidentally have a default here?
```suggestion
	environment: string
```